routes/web.php

```diff
@@ -32,5 +32,6 @@
 Route::group(['prefix' => 'admin', 'as' => 'admin.'], function () {
     Route::get('/attribute/edit', [AttributesController::class, 'editAttribute'])->name('admin.attributes.edit');
```

Comment:
Fix this one also, `->name('admin.attributes.edit');` is redundant. The route is already wrapped within the group `admin` but the name `->name('admin.attributes.edit');` repeats the `admin` prefix. It will cause the route to look something like this `admin.admin.attributes.edit`

Reply:
@KobeSergio 
Noted on this. I will revise like this:
From `name('admin.attributes.edit') `
To `name('attributes.edit') `